packages/utils/src/types.js

```diff
@@ -1,28 +0,0 @@
-// @flow
-import type { StyleSheet } from '@emotion/sheet'
-
-export type RegisteredCache = { [string]: string }
-
-export type Interpolation = any
```

Comment:
I see that we're also creating it's own `Interpolation` type instead of relying from `@emotion/serialize`

Reply:
This can be disregard as this is a mistake on my part